docs/known_issues.rst

```diff
@@ -125,6 +125,21 @@ Both will throw an exception (albeit not the expected UnitsError), if units do n
 
 See: https://github.com/astropy/astropy/issues/7582
 
+When broadcasting Quantities, it is necessary to pass `subok=True`, or else
+a bare `~numpy.ndarray` will be returned::
+
+   >>> q = u.Quantity(np.arange(10.), u.m)
+   >>> b = np.broadcast_to(q, (2, len(q)))
+   >>> b
+   array([[0., 1., 2., 3., 4., 5., 6., 7., 8., 9.],
+       [0., 1., 2., 3., 4., 5., 6., 7., 8., 9.]])
```

Comment:
Indentation.

Reply:
I put it this way because I think it would matter for the doctest, but it turns out that this entire file is not doctested anyway.